BasiliskII.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.haxxed.BasiliskII</id>
+
+  <name>Basilisk II</name>
+  <summary>A 68k Macintosh emulator</summary>
+
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-2.0-or-later</project_license>
+  <content_rating type="oars-1.1" />
+
+  <description><p>
+Basilisk II is an Open Source 68k Macintosh emulator. That is, it enables you to run 68k Mac OS software on you computer, even if you are using a different operating system. However, you still need a copy of Mac OS 6.x-8.1 and a Macintosh ROM image to use Basilisk II.
+  </p></description>
+
+  <url type="homepage">https://www.emaculation.com/doku.php/basilisk_ii</url>
+
+  <launchable type="desktop-id">com.haxxed.BasiliskII.desktop</launchable>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://pbs.twimg.com/media/En4E_C2VkAAojiG?format=png&amp;name=4096x4096</image>
```

Comment:
Screenshot url seems to return an empty page.

Reply:
correct url is https://pbs.twimg.com/media/En4E_C2VkAAojiG?format=png&name=4096x4096

Reply:
Build fails if I change this. TBH it's a placeholder anyway.

`
Error loading AppData file: failed to parse /app/share/appdata/net.cebix.basilisk.appdata.xml: Error on line 22: Entity did not end with a semicolon; most likely you used an ampersand character without intending to start an entity — escape ampersand as &amp;
`